semspaces/space.py

```diff
@@ -10,9 +10,9 @@
 try:
     import pandas as pd
 except ImportError:
-    print 'Warning: pandas not available. Exporting to pandas will not work.'
-
-import space_io
+    print('Warning: pandas not available. Exporting to pandas will not work.')
+# relative import in python 3 - remove references to space_io
+from .space_io import *
```

Comment:
Is there a reason why you chose to import elements of space_io into global namespace? I would prefer to keep it under space_io namespace.